pom.xml

```diff
@@ -312,6 +297,23 @@
             </exclusions>
         </dependency>
 
+        <dependency>
```

Comment:
This part shouldn't be necessary. The httpclient exclusion was removed in c7562dbd83434da953ead82ee30ba6622c4105bf and `KMSClientProvider` seems fine in IntelliJ after that (in relation to `URIBuilder`). It also uses a different version that the one that Hadoop depends on, which could cause problems.